File: Assignments/Recurssion/3.java
public class MaxValueInArray {
    public static void main(String[] args) {
        int[] array = {13, 1, -3, 22, 5};

        int maxValue = findMaxValue(array);

        System.out.println("The maximum value in the array is: " + maxValue);
    }

    public static int findMaxValue(int[] array) {
        if (array == null || array.length == 0) {
            throw new IllegalArgumentException("Array is empty or null");
        }

        int max = array[0];

        for (int i = 1; i < array.length; i++) {
            if (array[i] > max) {
                max = array[i];
            }
        }

        return max;
    }
}

File: Assignments/Recurssion/3.py
def find_max_value(arr):
    if not arr:
        raise ValueError("Array is empty")
    max_val = arr[0]
    for val in arr[1:]:
        if val > max_val:
            max_val = val
    return max_val

# Example usage
array = [13, 1, -3, 22, 5]
result = find_max_value(array)
print("The maximum value in the array is:", result)
